ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/Debian.yml

```diff
@@ -59,7 +59,7 @@ OpenJFX_Build_Tool_Packages:
   - libxxf86vm-dev
   - ruby
 
-gcc_7:                            # On all architectures but arm32
+gcc_7:                            # On all architectures but arm32 and riscv64
```

Comment:
Another situation where I feel we need a clean-up operation ... Ideally this should really depend on the distro and whether `gcc-7` is available, not fixed per architecture if possible, but not in scope for this PR.